addons/Points_To_Plot/src/presenter.js

```diff
@@ -55,6 +103,7 @@ function AddonPoints_To_Plot_create() {
 
         return points;
     };
+
     presenter.setShowErrorsMode = function() {
```

Comment:
Check Answers nie wyłącza SA. 
Użyj "isShowAnswersActive && presenter.hideAnswers()" lub "presenter.hideAnswers()"

Objawia się to dwoma błędami:
1. Kliknięcie SA a potem Check answers wyświetla punkty z SA.
2. Po wykonaniu błędu numer 1 wyłączenie SA powoduje zniknięcie odpowiedzi użytkownika do momentu ponownego włączenia Check answers. Możliwe, że ten błąd jest bardziej skomplikowany.